aten/src/ATen/native/README.md

```diff
@@ -384,6 +390,72 @@ will be automatically differentiated! This can be the case if the function imple
 only calls other operations which are themselves differentiable.  In this
 case, you don't have to write an entry in `tools/autograd/derivatives.yaml`.
 
+### Choosing the right dispatch keyword
+
+After writing a native function in C++, it's important to think about which dispatch keyword
+to use in native_functions.yaml as it gives dispatcher information about backend and autograd support
+of the implementation.
+
+Here're steps to follow to decide the right dispatch keyword:
+
+1. Think about inference, does your kernel work for all backends?
+
+    - No: you're likely providing different kerenls for different backends, e.g. DispatchStub or
```

Comment:
kernels

Reply:
BTW, the fact that DispatchStub doesn't work for all backends is non-obvious and probably worth explicitly calling out

Reply:
+1